helm_deploy/values-preprod.yaml

```diff
@@ -20,6 +20,7 @@ generic-service:
     API_BASE_URL_COURT_SENTENCING: https://remand-and-sentencing-api-preprod.hmpps.service.justice.gov.uk
     API_BASE_URL_ADJUDICATIONS: https://manage-adjudications-api-preprod.hmpps.service.justice.gov.uk
     API_BASE_URL_NON_ASSOCIATIONS: https://non-associations-api-preprod.hmpps.service.justice.gov.uk
+    API_BASE_URL_FINANCE: https://prisoner-finance-poc-api-preprod.hmpps.service.justice.gov.uk
```

Comment:
this and prod dont exist yet, so maybe point to self so that health check works?